nncf/dynamic_graph/graph_builder.py

```diff
@@ -105,9 +107,26 @@ def build_graph(self, model: torch.nn.Module, context_to_use: Optional['TracingC
                     self.custom_forward_fn(model)
         model.load_state_dict(sd)
 
+        graph = context_to_use.graph
+        self.set_module_details(model, graph)
         if isinstance(model, PostGraphBuildActing):
             model.post_build_graph_actions()
-        return context_to_use.graph
+        return graph
+
+    def set_module_details(self, model: torch.nn.Module, graph: 'NNCFGraph'):
+        general_conv_types = [v.op_func_name for v in NNCF_GENERAL_CONV_MODULES_DICT]
+        for node in graph.get_nodes_by_types(general_conv_types):
+            scope = node.op_exec_context.scope_in_model
+            module = get_module_by_scope(model, scope)
+            nx_node = graph.find_node_in_nx_graph_by_scope(scope)
+            module_details = {
+                "stride": module.stride,
+                "groups": module.groups,
+                "in_channels": module.in_channels,
+                "out_channels": module.out_channels,
+                "weight_requires_grad": module.weight.requires_grad,
```

Comment:
Use a dedicated structure with fixed fields instead of the dict. Update type hints accordingly.

Reply:
Done